assets/component-cart-notification.css

```
@@ -21,6 +21,12 @@
 }
 
 @media screen and (min-width: 750px) {
+  .header-wrapper:not(.header-wrapper--border-bottom)
+    + cart-notification
+    .cart-notification {
+    border-top-width: 0.1rem;
+  }
```

```suggestion
  .header-wrapper:not(.header-wrapper--border-bottom) + cart-notification .cart-notification {
    border-top-width: 0.1rem;
  }
```

I find this easier to read, but I'm not strongly opinionated on the matter.

---

Same. For a second I thought this was using some kind of nesting haha.

---

I made the changes. It was Prettier VS code plugin who does the autoformatting so I disabled it.